manifests/plugin/pulp.pp

```diff
@@ -2,47 +2,49 @@
 #
 # This class installs pulp plugin
 #
-# === Parameters:
+# === Advanced parameters:
```

Comment:
While I'm in here, cleaning this up. Sorry for the hideous diff. 

Every param in pulp.pp is advanced, a user never needs to do anything with this.